tests/css-epub-hyphens/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,20 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>EPUB CSS Hyphens</title>
+  </head>
+  <body>
+    <h1>-epub-hyphens Test</h1>
+
+   <p>1. This test passes if no hyphens appear within the following block of text.</p>
+
+   <p style="margin: 1em; -epub-hyphens: none;">The nineteenth century dislike of romanticism is the rage of Caliban not seeing his own face in a glass. The moral life of man forms part of the subject-matter of the artist, but the morality of art consists in the perfect use of an imperfect medium. No artist desires to prove anything. Even things that are true can be proved. No artist has ethical sympathies. An ethical sympathy in an artist is an unpardonable mannerism of style. No artist is ever morbid. </p>
+
+   <p>2. This test passes if hyphens appear in the any of these words in the following block of text: artist, express, everything, language, instruments, virtue, materials, musician, feeling, surface, or symbol.</p>
+
+   <p style="margin: 1em; -epub-hyphens: manual;">The art&shy;ist can ex&shy;press every&shy;thing. Thought and lang&shy;uage are to the art&shy;ist instru&shy;ments of an art. Vice and vir&shy;tue are to the art&shy;ist mater&shy;ials for an art. From the point of view of form, the type of all the arts is the art of the music&shy;ian. From the point of view of feel&shy;ing, the actor’s craft is the type. All art is at once sur&shy;face and sym&shy;bol.</p>
```

Comment:
Annoying nit that I don't expect you to do anything about: Since all (?) of these CSS properties have non-prefixed versions, and the browser cores ~all support the non-prefixed versions by this point, technically in order to test the prefixed default value, you need to *unset* the non-prefixed default to prove that changing the prefixed default did anything.

Like, for hyphens, the default is `manual`, so to prove that -epub-hyphens: manual is doing anything, I think you need this:
style="margin: 1em; hyphens: none; -epub-hyphens: manual;"

Obviously that's tedious, and since every prefixed property has multiple values, I think we can assume that if all but the default work, the default works too, but technically ...